package.json

```
@@ -4,6 +4,7 @@
   "private": true,
   "scripts": {
     "build": "pnpm -- turbo run build --filter=docs",
+    "build:turbo": "cd cli && make && cd ..",
```

I would do: 

```suggestion
    "build:turbo": "pnpm --filter=cli build",
```

Simpler and avoids the `cd`

---

good idea! done